package/kedro_viz/launchers/jupyter.py

```diff
@@ -12,7 +12,7 @@
 
 import IPython
 import requests
-from IPython.core.display import HTML, display
+from IPython.display import HTML, display
```

Comment:
Why did this change?

Reply:
new IPython version update - https://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html